node/internal.ts

```diff
@@ -243,7 +243,11 @@ export function _getVariable(name: string): string | undefined  {
     }
     else {
         // get the public value
-        varval = process.env[key];
+        if (_startsWith(key, 'VSTS_TASKVARIABLE_')) {
+            varval = _knownVariableMap[key].value
```

Comment:
Could you please add null check here - so undefined value will be returned for this case, but not an exception being thrown? (the same as for  process.env[key])